src/main/java/gregtech/common/blocks/wood/BlockPowderBarrel.java

```diff
@@ -0,0 +1,117 @@
+package gregtech.common.blocks.wood;
+
+import gregtech.api.block.IStateHarvestLevel;
+import gregtech.api.block.VariantBlock;
+import gregtech.common.entities.PowderbarrelEntity;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.Explosion;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+
+import javax.annotation.Nonnull;
+
+public class BlockPowderBarrel extends VariantBlock<BlockPowderBarrel.BlockType> {
```

Comment:
Do you need to extend VariantBlock if this has only one variant? Couldn't you just extend Block?